### ISSUE TITLE
Add section context menu to update all feeds in the section

### ISSUE THREAD

**Issue by auouymous** [gpodder/gpodder#1017]
Requested by Steve Silverwood on mailing list.

It should be possible to add a context menu to section labels in the channel list. An `Update podcasts` item could update all channels in the section. `Mark episodes as old` and `Archive` items could also exist, but other items probably shouldn't.

**Comment by elelay** [comment]
yes it could work. I had in mind adding this when rendering section headers selectable :+1:

**Comment by kb6ojs** [comment]
Thanks, glad to hear it.  I'd like to expand this by suggesting that:
1) the above also include renaming a section so that all feeds within that section are reassigned, rather than having to do it one feed at a time; and
2) possibly also allowing more than one feed to be selected at a time, and allowing "update" and reassigning to different sections.

Thanks.  //Steve//

**Comment by Snippet24816** [comment]
Myabe would be easier to implement to just add that a button for each category in a menu bar  rather than right clicking categories or selecting multiple of them in podcast list?

**Comment by kb6ojs** [comment]
I like this idea even better than the context-menu suggestion.  Click on “Category” menu, select the category from a drop down, and that action would list all options for the contents of that group.  (A “rename” option in this idea would change the category of all of the subscriptions in said section, which in large ones is a PITA and rarely needed, but every once in a while….)

 

Thanks!

 

//Steve//

 

Steve Silverwood, KB6OJS

Corona, CA USA

 ***@***.***> ***@***.***
